src/adaptation/massmatrix.jl

```diff
@@ -29,7 +29,7 @@ UnitMassMatrix() = UnitMassMatrix{Float64}()
 
 Base.string(::UnitMassMatrix) = "I"
 
-Base.resize!(pc::UnitMassMatrix, θ::AbstractVecOrMat) = nothing
+Base.resize!(pc::UnitMassMatrix, size_θ::Tuple) = nothing
```

Comment:
This still should not be `Base.resize!` - to confirm with its API and expected behaviour the second argument must be an integer.

Reply:
The main issue here is the naming of the resizing of mass matrix adaptor, extending `Base.resize!` and allowing its second argument to be a non-integer is not a good idea, and according to @yebai, we should have a consistent naming, so to keep it simple and concise, I think maybe we can change to `resize_adaptor!(pc::WelfordVar, size::Tuple)`?

Reply:
Sure, happy with `resize_adaptor!`.